cmd/clusterctl/client/cluster/upgrader.go

```diff
@@ -49,6 +49,10 @@ type UpgradePlan struct {
 	Providers    []UpgradeItem
 }
 
+const (
+	v1alpha4Contract = "v1alpha4"
```

Comment:
you can use `clusterv1.GroupVersion.Version` instead of this const, so the code will be more future proof